src/docs/wtperf.dox

```diff
@@ -133,29 +133,29 @@ number of async worker threads
 @par checkpoint_interval (unsigned int, default=120)
```

Comment:
It's probably worth making a visual check of this page in the browser (if you haven't already) just to make sure everything looks as expected.

Reply:
Looks good!